README.md

```diff
@@ -9,6 +9,17 @@ setupDevice(
                 fw: path.resolve(__dirname, 'fw/rssi-10040.hex'),
                 fwVersion: 'rssi-fw-1.0.0',
                 fwIdAddress: 0x2000,
+            },
+            custom: {
```

Comment:
Unfortunately the idea is that the key here (custom) is not allowed, this is used to identify the family (nrf51 or nrf52). Do we need something else here?

Reply:
Ok, then my assumption regarding the attribute was wrong. I just wanted to show an example on how to use the validator. I'll update the PR.